File: API/APIMeow/ViewModels/Edit/EditTransacaoViewModel.cs
public class EditTransacaoViewModel
{
    public int IdTransacao { get; set; }
    public string Nome { get; set; }
   public decimal QuantiaDinheiro { get; set; }
    public DateTime DataCriacao { get; set; }
    public char Feita { get; set; } = 'N';
    public DateTime DataFinalizacao { get; set; }
    public int IdClassificacao { get; set; }
    public int? IdRecorrencia { get; set; }
    public int? IdMeta { get; set; }
    public int? IdCofrinho { get; set; }
}

File: front_meow/lib/services/ViewModel/EditTransacaoViewModel.dart
class EditTransacaoViewModel {
  int idTransacao;
  String nome;
  double quantiaDinheiro;
  DateTime dataCriacao;
  String feita;
  DateTime dataFinalizacao;
  int idClassificacao;
  int? idRecorrencia;
  int? idMeta;
  int? idCofrinho;

  EditTransacaoViewModel({
    required this.idTransacao,
    required this.nome,
    required this.quantiaDinheiro,
    required this.dataCriacao,
    this.feita = 'N',
    required this.dataFinalizacao,
    required this.idClassificacao,
    this.idRecorrencia,
    this.idMeta,
    this.idCofrinho,
  });

  Map<String, dynamic> toJson() => {
    'IdTransacao': idTransacao,
    'Nome': nome,
    'QuantiaDinheiro': quantiaDinheiro,
    'DataCriacao': dataCriacao.toIso8601String(),
    'Feita': feita,
    'DataFinalizacao': dataFinalizacao.toIso8601String(),
    'IdClassificacao': idClassificacao,
    'IdRecorrencia': idRecorrencia,
    'IdMeta': idMeta,
    'IdCofrinho': idCofrinho,
  };
}
